src/Twilio/Converters/PhoneNumberConverter.cs

```diff
@@ -17,7 +17,9 @@ public class PhoneNumberConverter : JsonConverter
         /// <param name="serializer">unsued</param>
         public override void WriteJson(JsonWriter writer, object value, JsonSerializer serializer)
         {
-            var t = JToken.FromObject(value.ToString());
+            var t = value.ToString() != null
```

Comment:
When this line is called, is object null?

Reply:
Object is not null here, but value.ToString() is null, as ToString() method is overriden.
Discussed [here](https://github.com/twilio/twilio-csharp/pull/632#discussion_r975579431) previously.